foreman-proxy/logrotate.conf

```diff
@@ -1,4 +1,7 @@
-/var/log/foreman-proxy/*log {
+/var/log/foreman-proxy/cron.log
```

Comment:
I'm not sure, but foreman-debug-proxy is set to collect it [here](https://github.com/theforeman/smart-proxy/blob/2cc33adc5e63fb0acf35f714ec3372c5ee555d3d/extra/foreman-debug-proxy#L11) so I thought it would be good to include it as well.